lib/containers/utils.pm

```diff
@@ -79,6 +79,49 @@ sub registry_url {
     return sprintf("%s/%s:%s", $repo, $container_name, $version_tag);
 }
 
+sub test_update_cmd {
+    my %args = @_;
+    my $runtime = $args{runtime};
+    my $container = $args{container};
+
+    # podman update was added to v4.3.0
+    if ($runtime eq 'podman') {
+        my $version = script_output "podman version | awk '/^Version:/ { print \$2 }'";
+        if (package_version_cmp($version, '4.3.0') <= 0) {
+            record_info("The update command is not supported on podman $version");
+            return;
+        }
+    } elsif ($runtime ne 'docker') {
+        die "Unknown $runtime";
+    }
```

Comment:
Careful, we test Docker on [15-SP3](https://openqa.suse.de/tests/10361400).

Reply:
> Careful, we test Docker on [15-SP3](https://openqa.suse.de/tests/10361400).

This test is only for a `$runtime` other than podman or docker.

Reply:
> I mean, are you sure that Docker on SLE 12-SP3 supports this feature?

Both 15-SP5 & 12-SP3 use 20.10.17 released last year, but just to be sure - due to kernel & systemd differences - I created a VR for 12-SP3:
http://1b124.qa.suse.de/tests/311#

Reply:
The VR passed the Docker stage.